### PR TITLE
Upgrade Sidebar

### PR DESCRIPTION
**Pull request by arsengit**
- Added aim version info, slack channel and docs quick links in sidebar
- Added slack icon

### CODE REVIEW

**Comment by arsengit** [comment]
resolves #1178

File: aim/web/ui/src/components/SideBar/SideBar.tsx [hunk 1]
```diff
@@ -47,6 +50,8 @@ function SideBar(): React.FunctionComponentElement<React.ReactNode> {
     }
   }
 
+  console.log(AIM_VERSION);
```
Comment (VkoHov) [inline aim/web/ui/src/components/SideBar/SideBar.tsx:53]: remove console.log

**Review by rubenaprikyan — APPROVED** [review]
lgtm

File: aim/web/ui/src/components/SideBar/SideBar.tsx [hunk 2]
```diff
@@ -92,6 +97,31 @@ function SideBar(): React.FunctionComponentElement<React.ReactNode> {
...
+              target='_blank'
+              href='https://aimstack.slack.com'
+              rel='noreferrer'
+              className='Sidebar__bottom__anchor'
+              onClick={() => trackEvent('[Sidebar] go to slack')}
+            >
+              <Icon name='slack' />
+            </a>
+          </Tooltip>
+          <Tooltip title='docs' placement='right'>
```
Comment (roubkar) [inline aim/web/ui/src/components/SideBar/SideBar.tsx:112]: maybe the tooltip should be capitalized heere? (`Docs`)

**Review by roubkar — APPROVED** [review]

File: aim/web/ui/src/components/SideBar/SideBar.tsx (hunk 1 above)
Comment (arsengit) [reply aim/web/ui/src/components/SideBar/SideBar.tsx:53]: oops

**Review by VkoHov — APPROVED** [review]

**Pull request merged**